reactor-client/src/main/java/com/aerospike/client/reactor/dto/KeysRecords.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2012-2018 Aerospike, Inc.
+ *
+ * Portions may be licensed to Aerospike, Inc. under one or more contributor
+ * license agreements WHICH ARE COMPATIBLE WITH THE APACHE LICENSE, VERSION 2.0.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.aerospike.client.reactor.dto;
+
+import com.aerospike.client.Key;
+import com.aerospike.client.Record;
+
+/**
+ * Container object for keys identifier and exists flags.
+ */
+public final class KeysRecords {
+	/**
+	 * Unique identifiers for records.
+	 */
+	public final Key[] keys;
+
+	/**
+	 * Records headers and bin data.
+	 */
+	public final Record[] records;
+
+
```

Comment:
Map<Key, Record> would look more organic in this case. it's a pity that client operates with arrays in interfaces :(

Reply:
Disagree. Usually we iterate over results instead of accessing by key. Also this will add additional computation. What about to introduce utility method `asMap()` in `KeysRecords`?